helm/cluster-operator-chart/templates/configmap.yaml

```diff
@@ -25,6 +25,9 @@ data:
     service:
       clusterService:
         address: 'http://cluster-service:8000'
+      image:
+        registry:
+          domain: '{{ .Values.Installation.V1.Registry.Domain }}'
```

Comment:
This should match. https://github.com/giantswarm/cluster-operator/blob/f03ca1b29eb40615afbc7da28964dc046a06804e/main.go#L119